app/src/test/java/com/codesoom/assignment/domain/DescribeToy.java

```diff
@@ -0,0 +1,34 @@
+package com.codesoom.assignment.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Nested
+@DisplayName("Toy 클래스는")
+class DescribeToy {
+    // TODO
+    // 1. getter test
+    // 2. setter test
+
+    @Nested
+    @DisplayName("getter가 호출되었을 경우")
+    class DescribeWithGetter {
```

Comment:
`with`는 `Context`에 사용해 주세요. 그리고 `getter`는 `경우`로 표현하기보다는 테스트 대상으로 취급하는 쪽이 좋겠습니다.
```suggestion
    class DescribeGetter {
```

Reply:
그렇네요. 읽어놓고도 놓쳤었네요.